lib/generators/solidus_starter_frontend/solidus_starter_frontend_generator.rb

```diff
@@ -29,9 +29,9 @@ def install
 
     # Text updates
     append_file 'config/initializers/assets.rb', "Rails.application.config.assets.precompile += ['solidus_starter_frontend_manifest.js']"
-    append_file 'vendor/assets/javascripts/spree/frontend/all.js', "//= require spree/frontend/solidus_starter_frontend\n"
-    inject_into_file 'vendor/assets/stylesheets/spree/frontend/all.css', " *= require spree/frontend/solidus_starter_frontend\n", before: %r{\*/}, verbose: true
     inject_into_file 'config/initializers/spree.rb', "require_relative Rails.root.join('lib/solidus_starter_frontend/config')\n", before: /Spree.config do/, verbose: true
     gsub_file 'app/assets/stylesheets/application.css', '*= require_tree', '* OFF require_tree'
+
+    invoke 'solidus_starter_frontend:vendor_assets'
```

Comment:
@waiting-for-dev Regarding https://github.com/nebulab/solidus_starter_frontend/pull/174#issuecomment-894131810, I was able to reuse the `vendor_assets` generator here using Thor's `invoke` method. I found out about `invoke` from https://stackoverflow.com/a/12029262/65925. I initially tried Rail's [`generate` method](https://guides.rubyonrails.org/generators.html#generate) but the `solidus_starter_frontend` wasn't able to find the generator. Please see the video below:

https://www.loom.com/share/c27a99356ccf41c4a99f092f0704d099

Reply:
Oh, I see. `generate` seems unavailable because `solidus_starter_frontend` is not in the `Gemfile`, so the Rails context can't pick up the generators it provides.

As long as `VendorAssetsGenerator` remains simple, it'll work. I took a look at the post on SO you shared, and another caveat is that `invoke` [wouldn't dispatch arguments if needed](https://stackoverflow.com/questions/5890650/how-to-get-generators-call-other-generators-in-rails-3#comment34496472_12029262).

What do you think if we add a comment with all of that so that we're not caught in the future?

Reply:
Sounds good. I'll add comments.